rules/falco_rules.yaml

```diff
@@ -395,10 +381,20 @@
 - rule: Unexpected outbound connection destination
   desc: Detect any outbound connection to a destination outside of an allowed set of ips, networks, or domain names
   condition: >
-    consider_all_outbound_conns and outbound and not
-    ((fd.sip in (allowed_outbound_destination_ipaddrs)) or
-     (fd.snet in (allowed_outbound_destination_networks)) or
-     (fd.sip.name in (allowed_outbound_destination_domains)))
+    consider_all_outbound_conns and outbound
+  exceptions:
```

Comment:
Mark, just out of curiosity, have you run some benchmarks to evaluate the performance changes?

I mean, expressing "exceptions" with this new first-class citizen is faster or slower than expressing them with the "not" operator in the condition.

Reply:
I imagine there's little (negligible) overhead during the parsing phase.

I don't expect performance to decrease at runtime since exceptions get translated...

Am I correct?

Reply:
The exceptions are folded into the condition string at the time the rules are loaded, so I'm not worried about the conversion overhead.

I think the resulting condition is as about as efficient as the current chain of macros are. Each macro effectively was one tuple of exception fields + values.

Reply:
Yeah, exactly what I was supposing. Thanks for confirming it Mark!